.travis.yml

```diff
@@ -1,7 +1,42 @@
 ---
-rvm:
-- 2.3.1
+sudo: false
 language: ruby
-bundler_args: "--without development integration"
-gemfile:
-- Gemfile
+bundler_args: "--without development"
+dist: trusty
+cache: bundler
+
+rvm: 2.3.3
+
+before_install:
+  - gem update --system # see https://github.com/bundler/bundler/issues/5357
+
+env:
+ - INSTANCE=ubuntu-12-04
+ - INSTANCE=ubuntu-14-04
+ - INSTANCE=ubuntu-16-04
+ - INSTANCE=centos-6
+ - INSTANCE=centos-7
+ - INSTANCE=debian-7
+ - INSTANCE=debian-8
+ - INSTANCE=fedora-24
+ - INSTANCE=fedora-25
+
+script:
+  -  bundle exec rake prepare_do_env kitchen
+
+matrix:
```

Comment:
@chris-rock could you maybe explain it? Maybe I miss something: via your suggestion it gets 3x-4x more configuration lines with exactly the same result. The "job definition" is a bit easier to understand, however `.travis.yml` is always a bit ugly/complicated in my eyes if I compare it for instance with `.gitlab-ci.yml` job definitions...